src/Command/RemoveDuplicateAssetCommand.php

```diff
@@ -215,7 +216,11 @@ private function findFirstValidAsset(array $assetIds)
             }
         }
 
-        throw new Exception("None of the duplicate assets have an associated file that actually exists");
+        $commaSeparatedIds = implode(', ', $assetIds);
+
+        $this->output->writeln("None of the duplicate assets($commaSeparatedIds) have an associated file that actually exists");
+
+        return null;
```

Comment:
@IronSean as part of the duplicate asset cleanup, should be removing the hash association to the asset so it doesn't come up anymore? Because it seems to be finding the asset, just not returning any data